### PR TITLE
Astronauts

### PR DESCRIPTION
**Pull request by maxbachmann**
@Psychokiller1888 
Can you take a  look at it? I guess it should be fine this way

### CODE REVIEW

File: PublishedModules/mjlill/Astronauts/Astronauts.py [hunk 1]
```diff
@@ -42,12 +42,15 @@ def onMessage(self, intent: str, session: DialogSession) -> bool:
 				if not amount:
 					text = managers.TalkManager.getrandomTalk('noAstronauts')
 				elif amount == 1:
-					astronauts = data['people'][0]['name']
-					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(astronauts)
+					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(
```
Comment (Psychokiller1888) [inline PublishedModules/mjlill/Astronauts/Astronauts.py]: It's correct but if you want to be fast you can use the helper self.randomTalk(talk)

File: PublishedModules/mjlill/Astronauts/Astronauts.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+				url = 'http://api.open-notify.org/astros.json'
+	
+				req = requests.get(url=url)
+				data = req.json
+	
+				amount = data['number']
+				if not amount:
+					text = managers.TalkManager.getrandomTalk('noAstronauts')
+				elif amount == 1:
+					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(
```
Comment (Psychokiller1888) [inline PublishedModules/mjlill/Astronauts/Astronauts.py]: It's correct but if you want to be faster you can use the helper self.randomTalk()

Comment (maxbachmann) [reply PublishedModules/mjlill/Astronauts/Astronauts.py]: ah ok will use that

Comment (maxbachmann) [reply PublishedModules/mjlill/Astronauts/Astronauts.py]: @Psychokiller1888 can you check whether I use it the correct way?

**Pull request merged**